tests/platform_tests/api/test_fan.py

```diff
@@ -128,6 +157,7 @@ def test_get_direction(self, duthost, localhost, platform_api_conn):
             direction = fan.get_direction(platform_api_conn, i)
             if self.expect(direction is not None, "Unable to retrieve Fan {} direction".format(i)):
                 self.expect(direction in FAN_DIRECTION_LIST, "Fan {} direction is not one of predefined directions".format(i))
+                self.compare_value_with_platform_facts('direction', direction, i)
```

Comment:
We have also decided not to store fan direction in platform.json, as it is not necessarily tied to the platform. Please remove this check.